### DISCUSSION TITLE
Add setting to change font just for timestamps so that you can use a monospaced font.

### DISCUSSION THREAD

**Discussion by mastondzn** [Chatterino/chatterino2#4966] [Ideas] [answered]
If you use a font that does not have monospaced numbers you get misaligned badges/names that look unpleasant:

![image](https://github.com/Chatterino/chatterino2/assets/22679886/a6acff6a-b6d0-459a-bc11-c0932cee00ce)

I propose that another font setting is introduced just for the timestamp so that you can pick a monospaced font instead of whatever font you have for messages.

**Comment by 8thony** [comment, marked as answer]
- Fixed since #6062
